main-site/src/pages/sponsor-us.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import type { HeadFC } from 'gatsby';
+import '../../../shared-ui/style/globals.css';
+import '../../../shared-ui/style/globals.css';
+import Header from '../../../shared-ui/components/header/Header';
+import { mainSiteTabInfo } from '../../../shared-ui/lib/data';
+import Footer from '../../../shared-ui/components/footer/Footer';
+import useMatchMedia from 'react-use-match-media';
+import { min } from '../../../shared-ui/lib/responsive';
+import { StyledPageContainer } from '../../../shared-ui/styled-components/Background.styles';
+import AboutSection from '../../components/about-section/AboutSection';
+import ExploreSection from '../../components/explore-section/ExploreSection';
+import EventsCalendarSection from '../../components/events-calendar-section/EventsCalendarSection';
+import TestimonialsSection from '../../components/testimonials-section/TestimonialsSection';
+
+const SponsorUsPage: React.FC = () => {
+  const [isDay, setIsDay] = useState<boolean>(true);
+  const isDesktop = useMatchMedia(min.tabletXs);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const getBackgroundClassName = (): string => {
+    if (isDay && isDesktop) {
+      return 'day-background-desktop';
+    }
+    if (isDay && !isDesktop) {
+      return 'mobile-light-background';
+    }
+    if (!isDay && isDesktop) {
+      return 'dark-background-desktop';
+    }
+    return 'mobile-dark-background';
+  };
+  useEffect(() => {
+    setIsLoading(false);
+  }, []);
+
+  if (isLoading) {
+    return null;
+  }
+
+  return (
+    <StyledPageContainer className={getBackgroundClassName()}>
+      <Header tabs={mainSiteTabInfo} isDay={true} />
+      <ExploreSection />
+      <AboutSection />
+      <EventsCalendarSection />
+      <TestimonialsSection />
+      <Footer tabs={mainSiteTabInfo} isDay={isDay} />
+    </StyledPageContainer>
+  );
+};
+
+export default SponsorUsPage;
+
+export const SponsorUsTitle: HeadFC = () => <title>Sponsor Us</title>;
```

Comment:
it doesn't look like this is properly routing to a new page right now? it seems to show the main page contents. unsure if you meant to do this within this PR or a follow-up PR

Reply:
I just copied in some filler components for now from the main page. It is routing it just as a few of the same components from the main page.